exchanges/execute/src/execute.test.ts

```diff
@@ -65,16 +65,17 @@ describe('on operation', () => {
     );
 
```

Comment:
Let's add a test for the functionality

Reply:
I've added a test to make sure `subscribe` is being called. Further tests are blocked by the eslint `es5/no-generators` rule, since I would probably need some sort of AsyncIterator to test this further:

```patch
diff --git a/exchanges/execute/src/execute.test.ts b/exchanges/execute/src/execute.test.ts
index f73d8737..092d2061 100644
--- a/exchanges/execute/src/execute.test.ts
+++ b/exchanges/execute/src/execute.test.ts
@@ -51,6 +51,11 @@ beforeEach(() => {
   jest.clearAllMocks();
   mocked(print).mockImplementation(a => a as any);
   mocked(execute).mockResolvedValue({ data: mockHttpResponseData });
+  mocked(subscribe).mockImplementation(async function* b() {
+    yield { data: { key: 'value1' } };
+    yield { data: { key: 'value2' } };
+    yield { data: { key: 'value3' } };
+  });
 });
 
 afterEach(() => {
@@ -133,6 +138,19 @@ describe('on operation', () => {
     });
   });
 
+  it('should return data from subscribe', async () => {
+    const context = 'USER_ID=123';
+
+    const responseFromExecuteExchange = await pipe(
+      fromValue(subscriptionOperation),
+      executeExchange({ schema, context })(exchangeArgs),
+      take(3),
+      toPromise
+    );
+
+    expect(responseFromExecuteExchange.data).toEqual({ key: 'value3' });
+  });
+
   it('should return the same data as the fetch exchange', async () => {
     const context = 'USER_ID=123';
 
diff --git a/exchanges/execute/src/execute.ts b/exchanges/execute/src/execute.ts
index 688fdf5e..29efc6ef 100644
--- a/exchanges/execute/src/execute.ts
```

Reply:
@Tigge You could build an async iterator manually, but that's obviously a bit of a waste of time 😄 So, feel free to disable the rule for tests by adding an exception here: https://github.com/FormidableLabs/urql/blob/c2b9eb5afc9088bda6668f80fff74677d4dbf95b/scripts/eslint/common.js#L64

Reply:
Thanks, more tests are now added. Or at least one more.